app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topicsControllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
we don't need this line yet, can you take it out until the code requires you to add it back in